### PR TITLE
fix bug :ln function return is wrong

### PR DESCRIPTION
**Pull request by YANGGMM**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #3934

## What this PR does / why we need it:
fix bug :ln function return is wrong

### CODE REVIEW

File: pkg/vectorize/momath/math.go [hunk 1]
```diff
@@ -100,7 +100,8 @@ func Ln(arg, result *vector.Vector) error {
 	for i, v := range argCol {
 		if !nulls.Contains(arg.Nsp, (uint64)(i)) {
 			if v <= 0 {
-				nulls.Add(result.Nsp, uint64(i))
+				//return error.
+				return errors.New("", "Logarithmic function input cannot be less than 0")
```
Comment (ouyuanning) [inline pkg/vectorize/momath/math.go:104]: better to use  moerr.

**Review by yingfeng — CHANGES_REQUESTED** [review]
do not contain any files except for source code, documents as well as test cases

File: pkg/vectorize/momath/math.go (hunk 1 above)
Comment (fengttt) [reply pkg/vectorize/momath/math.go:105]: less than?   How about 0?   Less than or equal to or non positive, 

Yes, please use moerr.

**Comment by YANGGMM** [comment]
prepare to re-pull

**Pull request closed without merging**